### PR TITLE
[LIBCLOUD-555] Added support for openstack compute availability zone

### PR DESCRIPTION
**Pull request by syndicut**
https://issues.apache.org/jira/browse/LIBCLOUD-555

### CODE REVIEW

File: libcloud/compute/drivers/openstack.py [hunk 1]
```diff
@@ -1318,6 +1318,9 @@ def _create_args_to_params(self, node, **kwargs):
                                                                  {}))
         }
 
+        if 'availability_zone' in kwargs:
+            server_params['availability_zone'] = kwargs['availability_zone']
```
Comment (Kami) [inline libcloud/compute/drivers/openstack.py]: This is an extension param so please prefix it with `ex_`:

``` python
 if 'ex_availability_zone' in kwargs:
    server_params['availability_zone'] = kwargs['ex_availability_zone']
```

Comment (Kami) [inline libcloud/compute/drivers/openstack.py]: Please also update `create_node` docstring and document this kwarg there - https://github.com/syndicut/libcloud/blob/LIBCLOUD-555_openstack_availability_zone/libcloud/compute/drivers/openstack.py#L1218

Comment (syndicut) [inline libcloud/compute/drivers/openstack.py]: Done!

**Comment by Kami** [comment]
Merged into trunk.

Thanks.

**Pull request closed without merging**